src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -138,4 +139,12 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Returns true if none of the prefixes contains empty {@code Optional} values in the given
+     * {@code ArgumentMultimap}.
+     */
+    public static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
```

Comment:
Dont see this used anywhere? am i missing smth?

Reply:
It was originally from `AddOrderCommand.java` as a private static method - I placed it in `ParserUtil` as I thought it's a nice-to-have util function for parsing, but I didn't end up using it. I'll add a commit to remove the addition